pages/blog/_slug.vue

```diff
@@ -150,16 +154,34 @@ export default {
   content: {
     liveEdit: false,
   },
-  async asyncData({ $content, params }) {
-    const article = await $content('blog', params.slug).fetch()
+  async asyncData({ params }) {
+    const response = await axios.get(
+      `${process.env.strapiUrl}/api/blogs?filters[slug][$eq]=${params.slug}&populate=*`
+    )
+    const readingStats = readingTime(response.data.data[0].attributes.body)
```

Comment:
should not be doing `response.data.data[0]` at multiple places. Rather set a variable blog = response.data.data[0] and then use that